packages/components/src/styles/link-button-pure-styles.ts

```diff
@@ -161,7 +161,8 @@ export const getLinkButtonPureStyles = (
       label: mergeDeep(
         buildResponsiveStyles(hideLabel, getVisibilityJssStyle),
         buildResponsiveStyles(alignLabel, (alignLabelValue: AlignLabel) => ({
-          order: alignLabelValue === 'left' ? -1 : 0,
+          // TODO: we should remove 'left' here and map the value in the component class already to 'start' but might be difficult due to breakpoint customizable prop value
+          order: ['left', 'start'].includes(alignLabelValue) ? -1 : 0,
```

Comment:
This creates a new untyped array with 2 elements, iterates over them and does the comparison each time this function is called. Better would be to just do the 2 comparisons.